contracts/governance/src/governance.rs

```diff
@@ -71,6 +76,7 @@ impl GovernanceContract {
     }
 
     fn is_allowed(&self, address: &PublicKey) -> Result<(), Error> {
+        #[allow(clippy::needless_option_as_deref)]
```

Comment:
consider instead fixing the clippy nit, if applicable. otherwise, put a comment explaining why it is not desirable to fix that nit